packages/react-components/src/components/Select/Select.tsx

```diff
@@ -123,8 +146,12 @@ export default function Select<T extends object>({
                 return "Select an item";
               }}
             />
+            {isInvalid && iconError}
```

Comment:
I like this inline icon addition to align better with the text fields. We should get the designs updated to reflect this, even though it's a rare state for a select menu.

<img width="446" alt="Screenshot 2024-07-17 at 1 33 22 PM" src="https://github.com/user-attachments/assets/9c31fb0b-0b97-4565-be87-18a03da88d30">